cmd/cloud-controller-manager/nodeipamcontroller.go

```diff
@@ -84,11 +82,6 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 		return nil, false, err
 	}
 
-	// failure: more than one cidr and dual stack is not enabled
-	if len(clusterCIDRs) > 1 && !utilfeature.DefaultFeatureGate.Enabled(features.IPv6DualStack) {
```

Comment:
dual stack has been GA for a long time, and this is deviating from the original implementation in

https://github.com/kubernetes/kubernetes/blob/83334ccaa1aebfd0089968c22fca5d9ee5ffb56e/cmd/cloud-controller-manager/nodeipamcontroller.go